examples/index_robust04.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+PISA_PREFIX=~/src/pisa
```

Comment:
Can you get PISA's folder with bash instead?

Reply:
When I started fiddling I couldn't get `make install` work out of the box (there were some dependency errors if I remember correctly), so I run the binaries from the building directory directly.

Reply:
The dependency was libtbb.so.2. #242